### PR TITLE
[Event Hubs Client] Tracks One and Two (Minor Cleanup and Tweaks)

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The intent of these changes is to address some minor clean-up items, such as a spell checking pass, a formatter pass, some minor tweaks to test and utility infrastructure.

# Last Upstream Rebase

Wednesday, August 21, 2019 3:52pm (EDT)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample06_PublishEventsToSpecificPartitions.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class Sample06_PublishEventsToSpecificPartitions : IEventHubsSample
         ///   A short description of the sample.
         /// </summary>
         ///
-        public string Description { get; } = "An introduction to publishing events, using aa Event Hub producer that is associated with a specific partition.";
+        public string Description { get; } = "An introduction to publishing events, using a Event Hub producer that is associated with a specific partition.";
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample06_PublishEventsToSpecificPartitions.cs]: whoops.   Should be "an".

File: sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ internal static class AmqpError
         public static readonly AmqpSymbol PublisherRevokedError = AmqpConstants.Vendor + ":publisher-revoked";
 
         /// <summary>Indicates that an operation was canceled by the Event Hubs service.</summary>
-        public static readonly AmqpSymbol OperationCancelledError = AmqpConstants.Vendor + ":operation-cancelled";
+        public static readonly AmqpSymbol OperationCancelledError = AmqpConstants.Vendor + ":operation-canceled";
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs]: That's a mistake.   Revert.

File: sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ namespace Azure.Messaging.EventHubs.Processor
     ///
     internal class PartitionPump
     {
-        /// <summary>The <see cref="EventHubRetryPolicy" /> used to verify whether an exception is retryable or not.</summary>
+        /// <summary>The <see cref="EventHubRetryPolicy" /> used to verify whether an exception is retriable or not.</summary>
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs]: Technically, I think that both spellings are correct, but the rest of the codebase uses `retriable` so I let the spell checker have its way.

File: sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Amqp/AmqpClientConstants.cs [hunk 4]
```diff
@@ -33,7 +33,7 @@ class AmqpClientConstants
         public static readonly AmqpSymbol PartitionNotOwnedError = AmqpConstants.Vendor + ":partition-not-owned";
         public static readonly AmqpSymbol EntityDisabledError = AmqpConstants.Vendor + ":entity-disabled";
         public static readonly AmqpSymbol PublisherRevokedError = AmqpConstants.Vendor + ":publisher-revoked";
-        public static readonly AmqpSymbol OperationCancelledError = AmqpConstants.Vendor + ":operation-cancelled";
+        public static readonly AmqpSymbol OperationCancelledError = AmqpConstants.Vendor + ":operation-canceled";
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Amqp/AmqpClientConstants.cs]: This is a mistake and should be reverted.

File: sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Primitives/TaskExtensions.cs [hunk 5]
```diff
@@ -27,7 +27,7 @@ public static void WaitAndUnwrapException(this Task task)
         /// </summary>
         /// <param name="task">The task. May not be <c>null</c>.</param>
         /// <param name="cancellationToken">A cancellation token to observe while waiting for the task to complete.</param>
-        /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was cancelled before the <paramref name="task"/> completed, or the <paramref name="task"/> raised an <see cref="OperationCanceledException"/>.</exception>
+        /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was canceled before the <paramref name="task"/> completed, or the <paramref name="task"/> raised an <see cref="OperationCanceledException"/>.</exception>
```
Comment (kinelski) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Primitives/TaskExtensions.cs]: Is this a mistake as well?

Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Primitives/TaskExtensions.cs]: Nah, that's just a comment.   The others were tokens that have to be in sync with the server values.

**Review by kinelski — APPROVED** [review]
LGTM.

**Pull request merged**